components/user-manager/org.wso2.carbon.user.mgt.ui/src/main/resources/web/user/user-mgt.jsp

```diff
@@ -81,6 +82,10 @@
     java.lang.String errorAttribute = (java.lang.String) session.getAttribute(UserAdminUIConstants.DO_USER_LIST);
 
     String claimUri = request.getParameter("claimUri");
+    if (claimUri == null || claimUri.length() == 0) {
```

Comment:
Fixed in the PR : https://github.com/wso2/carbon-identity/pull/937 

Added StringUtils.isEmpty()